docs/circuit_cutting/explanation/index.rst

```diff
@@ -10,39 +10,182 @@ Circuit cutting is a technique to increase the size of circuits we can run on qu
+The sampling overhead is the factor by which the overall number of shots must increase for the quasiprobability decomposition to result in the same amount of error, :math:`\epsilon`, as one would get by executing the original circuit.
+The overhead of a cut gate depends on which gate is cut; see the final appendix of [`1 <https://arxiv.org/abs/2205.00016>`__] for details.
+For instance, a single cut CNOT gate incurs a sampling overhead of 9 [`2 <https://arxiv.org/abs/1909.07534>`__,\ `6 <https://arxiv.org/abs/2312.11638>`__].
+A circuit with :math:`n` wire cuts incurs a sampling overhead of O(:math:`16^n`) when classical communication is not available (LO setting); this is reduced to O(:math:`4^n`) when classical communication is available (LOCC setting) [`4 <https://arxiv.org/abs/2302.03366>`__].
+However, wire cutting with classical communication (LOCC) is not yet supported in CKT (see issue `#264 <https://github.com/Qiskit-Extensions/circuit-knitting-toolbox/issues/264>`__).
 
-As mentioned above, the cost of any simulation based on quasiprobability distribution is an exponential sampling overhead. The overhead of a cut gate depends on which gate is cut; see the final appendix of [`1 <https://arxiv.org/abs/2205.00016>`__] for details.  Here, we will focus on the CNOT gate.  If no real-time classical communication is available between qubits of the cut gate or wire, cut CNOT gates incur a sampling overhead of O(:math:`9^n`), and wire cuts incur a sampling overhead of O(:math:`16^n`), where :math:`n` is the total number of cuts. If real-time communication is available (i.e., if the hardware supports “dynamic circuits”), the sampling overhead for both CNOT gate and wire cuts may be reduced to O(:math:`4^n`) [`1 <https://arxiv.org/abs/2205.00016>`__,\ `4 <https://arxiv.org/abs/2302.03366>`__]; however, support for circuit cutting with classical communication (LOCC) is not yet supported in CKT.
+The QPD can be given explicitly as follows:
+
+.. math::
+   :label: eq:qpd
+
+   \mathcal{U} = \sum_i a_i \mathcal{F}_i ,
+
+where :math:`\mathcal{U}` is the channel implementing the desired operation, and each :math:`a_i` is a real coefficient corresponding to a quantum channel, :math:`\mathcal{F}_i`, that is realizable on hardware.
+
+Note that because this is a sum of channels, *not* a sum of unitaries, expectation values can be evaluated efficiently [`2 <https://arxiv.org/abs/1909.07534>`__].
+
+Results equivalent to original unitary channel can be obtained by a post-processing method, given the coefficients :math:`a_i` and each the outcome of each experiment corresponding to the different local (:math:`\mathcal{F}_i`) channels.  This post-processing boosts the magnitude of each measurement outcome by :math:`\sum_i \left| a_i \right|`, resulting in a sampling overhead of that quantity squared [`6 <https://arxiv.org/abs/2312.11638>`__]:
```

Comment:
```Suggestion
Results equivalent to original unitary channel can be obtained by a post-processing method, given the coefficients :math:`a_i` and the outcome of each experiment corresponding to the different local (:math:`\mathcal{F}_i`) channels. This post-processing boosts the magnitude of each measurement outcome by :math:`\sum_i \left| a_i \right|`, resulting in a sampling overhead of that quantity squared [`6 <https://arxiv.org/abs/2312.11638>`__]:

```

Reply:
Doesn't a measurement correspond to _a_ _particular_ channel? If so, wouldn't it be more accurate to say that magnitude of each measurement (i.e the measurement corresponding to channel (`\mathcal{F}_i)`) is boosted by `a_{i}`? It's the sum over `a_{i}` that I am not sure about.

Reply:
It's not so much that the channel gets boosted; it's that the classical outcome of +1 or -1 (which results from the measurement) gets boosted by $\sum_i \left| a_i \right|$ during reconstruction.  I borrowed this language from the recent paper, cited at the end of the line.